tools/CheckFiles.sh

```diff
@@ -43,6 +43,7 @@ tmpl_debugging_test() {
     test_check fail foo.cpp '#include "Utilities/TmplDebugging.hpp"'$'\n'
 }
 
+# Note: new tests should be added here and after `run_checks`
```

Comment:
Not usually.  Only the Travis-only checks go in those places.  The others go in `standard_checks` (and are defined in a different file).

Also, if we're going to have many of these we should put them in an array and use it in both places so there is nothing to keep in sync.

Reply:
what do you want the array to be called?

Reply:
I don't have a strong preference.  Maybe `ci_checks`?